interface_text_printer.go

```diff
@@ -35,4 +35,9 @@ type TextPrinter interface {
 	// Spaces are always added between operands and a newline is appended.
 	// It returns the number of bytes written and any write error encountered.
 	Printfln(format string, a ...interface{}) *TextPrinter
+
+	// PrintOnError prints every error which is not nil.
+	// If every error is nil, nothing will be printed.
+	// This can be used for simple error checking.
+	PrintOnError(a ...interface{})
```

Comment:
Hey @floaust, I think we should return TextPrinter everyhere, to make it fit to the rest of the interface.

```suggestion
	PrintOnError(a ...interface{}) *TextPrinter
```